examples/fast_neural_style/neural_style.py

```diff
@@ -0,0 +1,291 @@
+# coding: utf-8
+from __future__ import print_function, division
+
+import argparse
+import os
+import sys
+import time
+import re
+
+import numpy as np
+import random
+from PIL import Image
+import torch
+from torch.optim import Adam
+from torch.utils.data import DataLoader
+from torchvision import datasets
+from torchvision import transforms
+
+from ignite.engine import Engine, Events
+from ignite.handlers import ModelCheckpoint, Timer
```

Comment:
If you are not using Timer, it seems, we can remove it

Reply:
Incorporated